contrib/cloudshell/cloudshell-installation.sh

```diff
@@ -4,5 +4,5 @@ sudo bash
 adduser prowler
 su prowler
 pip install prowler
-cd /tmp || exit
+cd /tmp
```

Comment:
Why are you removing the `|| exit` is better to have that not to break anything in such a case that is not present.

Reply:
/tmp is present always in this case, it is not harmful at all to leave it or remove it